### PR TITLE
added list sorting options

### PR DESCRIPTION
**Pull request by brycehendrix**
This adds sorting options to the list command which is extremely useful in services with many secrets.

### CODE REVIEW

File: cmd/list.go [hunk 1]
```diff
@@ -76,3 +94,23 @@ func key(s string) string {
 	secretKey := tokens[1]
 	return secretKey
 }
+
+type ByName []store.Secret
```
Comment (dfuentes) [inline cmd/list.go]: This seems unused, we should remove it if that's the case.

Comment (brycehendrix) [reply cmd/list.go]: It's required by the 3 lines which follow it.

Comment (dfuentes) [reply cmd/list.go]: ah, I missed line 58, didn't realize that you always sort by name first.

**Comment by brycehendrix** [comment]
Not sure what the PR process here is- the PR was approved 2 weeks ago, but I don't have access to merge. Will it be merged as the next release is prepared?

**Comment by nickatsegment** [comment]
Sorry for the delay! Thanks for your contribution.

**Comment by nickatsegment** [comment]
Looks like the merge broke the build (probably bitrot); fixing.

**Comment by nickatsegment** [comment]
Huzzah! https://circleci.com/workflow-run/aed9eb98-35cf-4704-ba3e-4646d1b86277

**Pull request merged**